pontoon/base/static/js/main.js

```diff
@@ -448,6 +448,54 @@ $(function() {
       .toUpperCase().indexOf(m[3].toUpperCase()) >= 0;
   };
 
+  // Profile menu
+  // Register handlers to prep django-browserid before binding menu.
+  django_browserid.registerWatchHandlers(function() {
+    function signIn() {
+      Pontoon.startLoader();
+      django_browserid.login().then(function(verifyResult) {
+        console.log(verifyResult);
```

Comment:
Ah ! I had used this to figure out what was going wrong. I guess I understood it now. Fixing it right away. :smile: